api/v1alpha1/hosted_controlplane.go

```diff
@@ -44,6 +44,14 @@ type HostedControlPlaneSpec struct {
 	Platform  PlatformSpec `json:"platform"`
 	DNS       DNSSpec      `json:"dns"`
 
+	// ServiceAccountSigningKey is a reference to a secret containing the private key
+	// used by the service account token issuer. The secret is expected to contain
+	// a single key named "key". If not specified, a service account signing key will
+	// be generated automatically for the cluster.
+	//
+	// +optional
+	ServiceAccountSigningKey *corev1.LocalObjectReference `json:"serviceAccountSigningKey,omitempty"`
```

Comment:
Would be nice if you can explain here the format of that "key". I assume it must be a RSA private key in PKCS#1 format and PEM encoded, but better to be explicit.

Reply:
Ack, will update in a follow-up